### PR TITLE
Reworked Pages

### PR DESCRIPTION
**Pull request by Towito**
Adjust page routing. All links properly route to their respective changes. Add "login" page created from prior split. Added elements from Edit/View Scenario page into home page to prompt editors to make or edit existing scenario. Create New Scenario Button now links to a "New_Scenario" page that prompts user to input title and description. "new_scenario.js" needs to be edited in order to hold state of user input and submission. Login page still requires better stylization.
Add react-bootstrap and dependent packages to this folder in order to make previously created code work properly.

In short, something that should've been several pull requests, probably.

### CODE REVIEW

**Review by coffincw — APPROVED** [review]
Looks good, but yeah in the future try to split up changes like this into separate features.
